### DISCUSSION TITLE
Is it support ipv6?

### DISCUSSION THREAD

**Discussion by fuergaosi233** [prisma/prisma#16273] [Q&A]
I am using ipv6 to connect my database.
If i use [::1] as host, engine will throw P1001 with error messeage
```
Can't reach database server at `[::]`:`3306`

Please make sure your database server is running at `[::]`:`3306`.
```
I can use mycli connect this db with [::1].

**Comment by nurul3101** [comment]
Hey @fuergaosi233 👋 

What database are you using? Prisma should work as expected even if the connection string uses ipv6.
Can you try enabling [debug environment variable](https://www.prisma.io/docs/concepts/components/prisma-client/debugging#setting-the-debug-environment-variable) and see if you get any more information?

**Reply by fuergaosi233** [reply]
![image](https://user-images.githubusercontent.com/22197568/202478206-29a7dc9d-4f6d-471e-8aa6-212fb416be2e.png)
<img width="991" alt="image" src="https://user-images.githubusercontent.com/22197568/202478299-8e70237e-88e4-4433-8669-ee9f05f7c71b.png">
I just use Studio to test, it will also have this error in my production environment.

**Reply by fuergaosi233** [reply]
I am using MySQL and this is my database string:
```
mysql://root:root@[::1]:3306/my_db
```

**Reply by nurul3101** [reply]
Does this connection string also throw error?
```
mysql://root:root@127.0.0.1:3306/my_db
```

**Reply by fuergaosi233** [reply]
<img width="617" alt="image" src="https://user-images.githubusercontent.com/22197568/202518313-d4fdaf93-5b4b-4719-aeb8-82fb2374874e.png">
I can assure you that it works

**Reply by nurul3101** [reply]
Awesome! Will this connection string work, or is it necessary for you to use [::1] format?

**Reply by fuergaosi233** [reply]
![image](https://user-images.githubusercontent.com/22197568/202636555-2f84ed13-0489-40a5-87ba-f0d4cc5f300d.png)
I will get the same error. And if I use CLI to connect, it will work successfully.
![image](https://user-images.githubusercontent.com/22197568/202636834-b690b6cd-b2a7-424e-b6b8-6da065b5f093.png)

**Reply by nurul3101** [reply]
Okay, but if you continue to use the 127.0.0.1 connection string, will it work for you or is it mandatory to use the [::1] format? If it is mandatory we can open a Bug report to investigate what the issue could be and get further insights from engineering team.

**Reply by fuergaosi233** [reply]
Yes I have to use IPV6 to continue, this is just a Demo for reappear problem, I don't think IPv6 should be a problem

**Reply by fuergaosi233** [reply]
Hey @nurul3101 , Have you opened new issues of bug reports?

**Reply by nurul3101** [reply]
Hey @fuergaosi233, can you create a bug report following this [link](https://github.com/prisma/prisma/issues/new?assignees=&labels=kind%2Fbug&template=bug_report.yml), you would be better able to add all the connection string formats that you tried which worked for you and the connection string formats which didn't work. Our engineering team is monitoring the bug reports, they will be able to provide their insights once the bug report is created.

**Reply by pantharshit00** [reply]
@fuergaosi233 did you get a chance to create the bug report? If yes, it would be great if you can link it here so that other people landing on this thread can find it.

**Reply by Fenntasy** [reply]
https://github.com/prisma/prisma/issues/18079

**Reply by nurul3101** [reply]
Thanks for creating the bug report 👍

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
